README.md

```diff
@@ -22,28 +87,41 @@ Contents
 npm install --save-dev @manta-style/cli
 ```
 
-You could also install it globally, which adds a command line tool `ms` to your system.
+To install globally
 
-### Plugins
+```
+npm install -g @manta-style/cli
+```
 
-Manta Style needs plugins to support different file types and generate mock data.
+This adds a command line tool `ms` to your system.
 
-The example of Quick Start below is using TypeScript. So first you might want to install TypeScript support to Manta Style.
+### Builders
+
+Manta Style supports generation of mock data from different typing systems via binding builders.
+
+In [Quick Start](#quick-start) we show an example using TypeScript. You will need to [install TypeScript](https://www.typescriptlang.org/index.html#download-links) _and_ add the Manta Style TypeScript builder:
```

Comment:
by right you don't need to install `typescript` in your project. if it's not the case, i'll create a bug ticket to myself.